eng/mgmt/Directory.Build.Mgmt.props

```diff
@@ -1,4 +1,10 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(RepoRoot)\tools\bootstrapTools\bootstrap.targets" />
   <Import Project="CI.Bootstrap.targets" Sdk="Microsoft.Internal.NetSdkBuild.Mgmt.Tools" Version="0.10.0" />
+
+  <PropertyGroup>
+    <SignAssembly>true</SignAssembly>
+    <DelaySign>true</DelaySign>
+    <AssemblyOriginatorKeyFile>$(pkg_root_RepoToolsDir)\MSSharedLibKey.snk</AssemblyOriginatorKeyFile>
```

Comment:
Wouldn't this be better
```<AssemblyOriginatorKeyFile>$(RepoEngPath)\MSSharedLibKey.snk</AssemblyOriginatorKeyFile>```

Reply:
Yes it would be better (assuming the keys are the same which I expect they are) and we should definitely clean this up but I went with the direct copy past of the properties from your nuget package until I could chat with you about your thought on how you want to do this. 

For now I want to unblock folks like the Search team and I will clean this up once we figure out the plan.